flink-table/flink-table-api-java/src/main/java/org/apache/flink/table/api/TableEnvironment.java

```diff
@@ -89,6 +92,235 @@ static TableEnvironment create(EnvironmentSettings settings) {
 		return TableEnvironmentImpl.create(settings);
 	}
 
+	/**
+	 * Creates a Table from given values.
+	 *
+	 * <p>Examples:
+	 *
+	 * <p>You can use a {@code row(...)} expression to create a composite rows:
```

Comment:
The reason why I didn't link to a `row` is that it will be a different method depending which API users use.
It's either `org.apache.flink.table.api.Expressions#row` or `org.apache.flink.table.api.ImplicitExpressionConversions#row`

The `TableEnvironment` is shared between java and scala. I don't have a strong opinion on this though. If you think it's better to link to the Java method I can do it.

Reply:
Make sense to me. I'm fine with the current solution.